plugins/plugins.public.yaml

```diff
@@ -0,0 +1,39 @@
+# This file defines public plugins to be installed via `loopinstall`.
+
+# Common plugin configuration
+defaults:
+  # The `-s` flag is added to strip debug information from the binary to reduce
+  # the binary size for releases.
+  # See: `go tool link -help`
+  goflags: "-ldflags=-s"
+
+plugins:
+  cosmos:
+    - moduleURI: "github.com/smartcontractkit/chainlink-cosmos"
+      # Git reference - can be a tag, branch, or commit hash
+      # If not specified, uses the latest version.
+      gitRef: "f740e9ae54e79762991bdaf8ad6b50363261c056" # 2025-02-07
+      installPath: "github.com/smartcontractkit/chainlink-cosmos/pkg/cosmos/cmd/chainlink-cosmos"
+      # These will be copied into /usr/lib in the container.
+      libs:
+        - /go/pkg/mod/github.com/!cosm!wasm/wasmvm@v*/internal/api/libwasmvm.*.so
+
+  feeds:
+    - moduleURI: "github.com/smartcontractkit/chainlink-feeds"
+      gitRef: "7cd0000951353740d733892420bfd6e1c8eb5841" # 2025-02-27
+      installPath: "github.com/smartcontractkit/chainlink-feeds/cmd/chainlink-feeds"
+
+  solana:
+    - moduleURI: "github.com/smartcontractkit/chainlink-solana"
+      gitRef: "8f68f1d0afeaf4b60a2af3e3e78773d2560cc21f" # 2025-04-17
+      installPath: "github.com/smartcontractkit/chainlink-solana/pkg/solana/cmd/chainlink-solana"
+
+  starknet:
+    - moduleURI: "github.com/smartcontractkit/chainlink-starknet/relayer"
+      gitRef: "9a780650af4708e4bd9b75495feff2c5b4054e46" # 2025-02-04
+      installPath: "github.com/smartcontractkit/chainlink-starknet/relayer/pkg/chainlink/cmd/chainlink-starknet"
+
+  streams:
+    - moduleURI: "github.com/smartcontractkit/chainlink-data-streams"
+      gitRef: "036bb568a69d7b7a841017ac72a068f06c50c218" # 2025-03-25
+      installPath: "github.com/smartcontractkit/chainlink-data-streams/mercury/cmd/chainlink-mercury"
```

Comment:
solana, data-streams, and feeds plugins were getting their plugin version automatically from the `go.mod` before whereas now they are set here.

Do we want to fail CI if there's a mismatch between the go.mod version and what we're instaling here?

Reply:
Will need to add a CI check here to fail if there's a mismatch to keep this in sync.

Reply:
https://github.com/smartcontractkit/chainlink/pull/17360